### PR TITLE
Home and Authentication Pages

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello,

I have created Home and Authentication pages.

Please review and suggest.

### CODE REVIEW

**Review by vrinda-mahajan — APPROVED** [review]
The UI looks clean. Code is also good. 👍

File: index.html [hunk 1]
```diff
@@ -7,5 +7,109 @@
...
+
+    <main>
+      <section class="section-intro flex-row-center border-bottom-1">
+        <div class="section-text pl-2">
+          <h2 class="mb-2">Answer fun questions and test your knowledge!</h2>
+          <a href="#explore" class="button primary explore-btn"
+            >Explore Quizzes</a
+          >
+        </div>
+        <img class="home-img img-responsive" src="/images/37561.jpg" />
```
Comment (govindappaarun) [inline index.html:51]: add loading="lazy" for image to improve page loading

File: index.html [hunk 2]
```diff
@@ -7,5 +7,109 @@
...
+              href="pages/rules.html"
+              class="button button-anchor border-radius-0 mt-0p5"
+              >Play Now</a
+            >
+          </div>
+
+          <div class="card-vertical card-shadow flex-column-center text-center">
+            <img
+              class="card-img-vertical img-responsive"
+              src="/images/question-mark-2.jpg"
```
Comment (govindappaarun) [inline index.html:74]: same image loading

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
+@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css");
+@import url("https://dazzle-ui-components.netlify.app/components/components.css");
```
Comment (govindappaarun) [inline styles.css:2]: you can load minified  version of your component file also using https://www.jsdelivr.com/

**Review by govindappaarun — APPROVED** [review]
- Overall looks good
- had skimmed for few minutes only

**Pull request merged**